demisto_sdk/commands/common/tests/integration_test.py

```diff
@@ -238,13 +238,16 @@ def test_id_valid_subtype(self, current, answer):
         validator = IntegrationValidator(structure)
         assert validator.is_valid_subtype() is answer
 
+    DEFUALT_ARGS_1 = [
```

Comment:
we had `DEFUALT_ARGS_2` but not `DEFUALT_ARGS_1`? 🤦

Reply:
🤯